src/dash/student_view/SearchBar.jsx

```diff
@@ -29,7 +29,7 @@ class DashSearchBar extends React.Component {
                     <option value="week">This week</option>
                 </select>
                 <span className="input-group-btn">
-                    <button onClick={() => this.props.handleSearchClicked(this.state.subject, this.state.time)} style={{animationDelay: '2s'}} className="animated tada col-xs-12 btn btn-default btn-lg search-btn" type="button">
+                    <button onClick={this.state.subject != null ? () => this.props.handleSearchClicked(this.state.subject, this.state.time) : ''} style={{animationDelay: '2s'}} className="animated tada col-xs-12 btn btn-default btn-lg search-btn" type="button">
```

Comment:
Do we need an error message if the field is null? We don't have that yet, correct?

Reply:
I just have it doing nothing when the field is null right now. I could add that though.